docs/spec/v1beta2/ocirepositories.md

```diff
@@ -121,6 +121,9 @@ static credentials are used for authentication, either with
 `spec.secretRef` or `spec.serviceAccountName`.
 If you do not specify `.spec.provider`, it defaults to `generic`.
 
+For a complete guide on how to set up authentication for cloud providers,
+see the integration [docs](/flux/integrations/).
```

Comment:
Please move this to OCIRepository v1 docs.

Reply:
Fixed, thanks!